### PR TITLE
Fix inventory page description

### PR DESCRIPTION
**Pull request by ShimShtein**

### CODE REVIEW

File: webpack/ForemanInventoryUpload/Components/PageHeader/components/PageDescription/__tests__/__snapshots__/PageDescription.test.js.snap [hunk 1]
```diff
@@ -5,16 +5,39 @@ exports[`PageDescription rendering render without Props 1`] = `
...
+            Automatic inventory upload
+          </strong>,
+        }
+      }
+    />
   </Text>
   <Text>
-    Enabling inventory uploads is required by subscription watch. For more information about subscription watch see link:
+    <FormattedMessage
+      defaultMessage="To upload the data for a specific organization manually, select an organization and click {restartButtonName}."
```
Comment (knortema) [inline webpack/ForemanInventoryUpload/Components/PageHeader/components/PageDescription/__tests__/__snapshots__/PageDescription.test.js.snap]: This should read "To manually upload the data..."

**Comment by ShimShtein** [comment]
Ready for another round

**Comment by knortema** [comment]
The Change looks good to me. Thanks!

**Review by knortema — APPROVED** [review]
The changes look good. Thanks!

**Pull request merged**